corehq/apps/app_manager/suite_xml.py

```diff
@@ -1382,6 +1372,17 @@ def details(self):
                                     )
                                     if d:
                                         r.append(d)
+            if module.fixture_select.active:
+                d = Detail(
+                    id='m{}_fixture_select'.format(module.id),
```

Comment:
This should be a function in `id_strings` e.g. `id_strings.fixture_detail(modue)`